package.json

```diff
@@ -26,5 +50,62 @@
     "singleQuote": true,
     "tabWidth": 2,
     "trailingComma": "es5"
+  },
+  "dependencies": {
+    "@apollo/client": "^3.8.0-alpha.8",
+    "@apollo/datasource-rest": "^5.0.2",
+    "@apollo/server": "^4.4.1",
+    "@apollo/subgraph": "^2.3.3",
+    "@radix-ui/react-context-menu": "^2.1.3",
+    "@radix-ui/react-dialog": "^1.0.3",
+    "@radix-ui/react-dropdown-menu": "^2.0.4",
+    "@radix-ui/react-popover": "^1.0.5",
+    "@radix-ui/react-progress": "^1.0.2",
+    "@radix-ui/react-slider": "^1.1.1",
+    "@radix-ui/react-tabs": "^1.0.3",
+    "@radix-ui/react-tooltip": "^1.0.5",
+    "@tanstack/react-table": "^8.7.9",
+    "@wry/equality": "^0.5.3",
+    "camelize": "^1.0.1",
+    "classnames": "^2.3.2",
+    "cors": "^2.8.5",
+    "date-fns": "^2.29.3",
+    "deepmerge": "^4.3.0",
+    "express": "^4.18.2",
+    "formik": "^2.2.9",
+    "graphql": "^16.6.0",
+    "graphql-subscriptions": "^2.0.0",
+    "graphql-ws": "^5.12.0",
+    "lodash": "^4.17.21",
+    "lucide-react": "^0.124.0",
+    "node-fetch": "^2.6.7",
+    "node-vibrant": "^3.1.6",
+    "prism-react-renderer": "^1.3.5",
+    "prism-themes": "^1.9.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-markdown": "^8.0.5",
+    "react-router-dom": "^6.8.2",
+    "remark": "^14.0.2",
+    "remark-parse": "^10.0.1",
+    "rxjs": "^7.8.0",
+    "sass": "^1.58.3",
+    "strip-markdown": "^5.0.0",
+    "ts-invariant": "^0.10.3",
+    "vite": "^4.1.4",
+    "web-vitals": "^2.1.4",
+    "ws": "^8.12.1"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
   }
 }
```

Comment:
To be honest, I have no idea if its considered good practice to put both client and server packages in a single `package.json`. Since this is a full stack app where the server serves the HTML for the client, this felt like an ok approach.